src/ofxBonjourPublisher.mm

```diff
@@ -74,3 +88,9 @@ - (void)dealloc {
                                                    port:port
                                                  domain:@(domain.c_str())] ? true : false;
 }
+
+bool ofxBonjourPublisher::setTextRecord(std::vector<std::pair<std::string, std::string>> key_values = {}) {
+    NSMutableDictionary * record = [NSMutableDictionary dictionary];
+    for (const auto & [key, value]: key_values) [record setValue:@(value.c_str()) forKey: @(key.c_str())];
```

Comment:
this is a feature from C++17.

please rewrite as C++11 like:

```cpp
    for(const auto &kv : key_values) [record setValue:@(kv.second.c_str()) forKey: @(kv.first.c_str())];
```

Reply:
hey! now that C++17 is OK, what about these? note a few more changes below.